lightningd/dual_open_control.c

```diff
@@ -2486,11 +2556,19 @@ static const struct json_command openchannel_bump_command = {
 	"Attempt to bump the fee on {channel_id}'s funding transaction."
 };
 
+static const struct json_command openchannel_next_feerate_command = {
+	"openchannel_next_feerate",
+	"channels",
+	json_openchannel_next_feerate,
+	"What's the next feerate for an RBF of this {channel_id}?"
+};
```

Comment:
Well, that's up to the caller isn't it?  They just need to know the current feerate, and they can calculate the granularity for themselves.  Shouldn't we just put this info in listpeers?

Reply:
Per proposed spec, we use an automatically increasing feerate. Every RBF attempt bumps the feerate by 25% from the last *successful* attempt.

We need to know the feerate to build a new psbt though, so we expose it via an RPC.

Reply:
I feel like you commented elsewhere that we can/should do this via `listpeers`'s channel listing, which I agree is much better than this approach!